core/controllers/ingest.py

```diff
@@ -214,3 +304,46 @@ def save_submission_file(excel_file, submission_id):
     submission.submission_file = excel_file.stream.read()
     db.session.add(submission)
     db.session.commit()
+
+
+def modify_schema(schema):
+    """
+    Modifies the schema by removing specified tables and adding "Submission ID" as the first column to specified tables.
+
+    :param schema: The original schema dictionary.
+    :return: The updated schema dictionary.
+    """
```

Comment:
I know I suggested this way of doing it but I've thought on it and I actually think it's more maintainable and less error prone to have 2 separate verbose validation schemas. 

- The main reason is that if we make changes to _the_ validation schema in the current setup - it may have unintended affects on R1. 
- It'll also be clearer what the validation logic is for each round if we give round 1 it's own verbose schema

Reply:
Starting with the current schema and just making these changes shouldn't take very long so probably worth doing IMO